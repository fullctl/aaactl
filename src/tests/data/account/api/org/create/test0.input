{
  "name": "Foo Bar Org",
  "slug": "foobar"
}
