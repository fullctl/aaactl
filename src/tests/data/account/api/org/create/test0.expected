{"data": [{"id": 5, "name": "Foo Bar Org", "label": "Foo Bar Org", "slug": "foobar", "selected": null, "personal": false, "is_admin": false, "is_default": false, "roles": ["Admin", "admin"], "all_roles": ["Admin", "Member", "admin"]}], "errors": {}, "profiling": {"queries": 0}}
